k9mail/src/main/java/com/fsck/k9/mailstore/StoreSchemaDefinition.java

```diff
@@ -577,6 +578,12 @@ private void upgradeDatabase(final SQLiteDatabase db) {
                 if (db.getVersion() < 52) {
                     addMoreMessagesColumnToFoldersTable(db);
                 }
+                if (db.getVersion() < 53) {
+                    /*
+                     * All messages are initialized to "no encrypted" even if encrypted.
+                     */
+                    db.execSQL("ALTER TABLE messages ADD COLUMN encryption_type TEXT default 'NONE'");
```

Comment:
is there a reason you chose TEXT for this field? I think an INT, linked to a bunch of constants or an enum (not sure which is preferred in k-9) would be the appropriate type here.